src/reference/01-General-Info/04-Bintray-For-Plugins.md

```diff
@@ -115,6 +111,28 @@ API key for future use.
 *NOTE: We have to do this before we can link our package to the sbt
 org.*
 
+### User Documentation
+
+You should add a few lines to your plugin's README file so users know how to modify their `project/plugins.sbt` in order to use your plugin.
+SBT plugins always use Ivy style patterns, so the resolver incantation is a bit verbose.
+As always, the first component in the resolver is just a human-friendly name for the resolver.
+
+The second component has two tokens that matter, set from keys in your plugin's `build.sbt`: `my-org-name` (set from the `bintrayOrganization` key) and `my-repo-name` (set from the `bintrayRepository` key).
+
+The `addSbtPlugin` statement that follows has three fields set from keys that are also specified in your plugin's `build.sbt`:
```

Comment:
This information took me 3 hours to figure out on my second attempt, after spending 2 hours and giving up my first attempt. It is probably obvious to an SBT expert, but newbies probably won't know it. Docs should be written for everyone, not just experts. 

If this information is spelled out in detail elsewhere, then a link to the detail would be better, followed by a concrete example at this point. Abstract examples and unspoken assumptions often leave less experienced readers wondering.